README.md

```diff
@@ -0,0 +1,168 @@
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [My currency] <a name="about-project"></a>
+
+
+**[My currency]** My currency it's a web application where you can check the comparison between a certain amount of USD and other currencies. You can also check every currency and see the comparison of that one with the others, you can also filter directly for a specific currency.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li>HTML5</li>
+    <li>CSS</li>
+    <li>JAVASCRIPT</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Tools</summary>
+  <ul>
+    <li>React</li>
+    <li>Redux</li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[React]**
+- **[Redux]**
+- **[SPA]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[Live demo](https://drive.google.com/file/d/1wgARiuPFl28gF5zTK8KJav7RX8-gHx7X/view?usp=sharing)
+[Render](https://my-currency.onrender.com)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+you can use this code as a base for your own project.
+
+### Prerequisites
+
+In order to run this project you need:
+Node.js
+
+
+### Setup
+
+git clone https://github.com/Alejandro-Bernal-M/my-currency.git
+
+
+### Install
+
+npm install
+
+### Usage
+
+npm start
+
+
+### Run tests
+
+Tests under construction
+
+
+### Deployment
```

Comment:
- [ ] To make your documentation be more professional and descriptive, I think it would be better if you fill both `### Run tests` and `### Deployment` sub sections with appropriate content for example `Run Tests` with a command of how to run your tests and `Deployment` with the server you used to deploy your project(i.e. [Render](https://render.com/))